### PR TITLE
docs:update pallet readme

### PR DESCRIPTION
**Pull request by stanly-johnson**
Detailed README file for all pallets currently implemented. Based on [substrate FRAME pallets](https://github.com/paritytech/substrate/tree/master/frame/balances).

### CODE REVIEW

File: pallets/foreign_asset/README.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
 # Foreign Asset Pallet
 
-Foreign assets are any off-chain crypto or fiat asset that an attester claims to have. Foreign assets can be minted on chain in a permission way if the attester is trusted(in a whitelist) or in a permissionless way(not supported yet) by locking some funds as safeguard.
+Foreign assets are any off-chain crypto or fiat asset that an attester claims to have. Foreign assets can be minted on chain in a permissioned way if the attester is trusted (in a whitelist) or in a permissionless way (not supported yet) by locking some funds as safeguard.
+
+## Terminology
+
+- Attestor: Any external user/company that provides liquidity to VLN. The permission to be an attestor will initially be restricted via a whitelist, but will be opened up in future to all users.
```
Comment (stanly-johnson) [inline pallets/foreign_asset/README.md:7]: Is this whitelisting implemented? or is it TBD?

File: primitives/README.md [hunk 2]
```diff
@@ -1,2 +1,40 @@
...
+}
+```
+
+#### Collateral
+
+Collateral lists all the cryptocurrencies that can be deposited to create assets onchain.
+
+```rust
+pub enum Collateral {
+        Usdc = "USDC",
```
Comment (stanly-johnson) [inline primitives/README.md:23]: Did we remove BTC? Also how will we recognise a deposit of USDC? 🤔

File: primitives/README.md [hunk 3]
```diff
@@ -1,2 +1,40 @@
...
+```rust
+pub enum Fiat {
+        Cop = "COP",
+        Vez = "VEZ",
+    }
+```
+
+#### Usdv
+
+This is the default asset of VLN.
```
Comment (stanly-johnson) [inline primitives/README.md:40]:
```suggestion
This is the default asset of VLN. Usdv is backed by a combination of crypto and fiat assets provided by the liquidity providers within VLN
```

File: pallets/backed_asset/README.md [hunk 4]
```diff
@@ -1,3 +1,30 @@
 # Backed Asset Pallet
 
-An asset that is backed by one or more similar assets keeping track of the share that each collateral holds.
+This pallet allows creation of asset that is backed by one or more similar assets keeping track of the share that each collateral holds. Any liquidity provider can create a backed asset by depositing collateral to this pallet.
+
+## Terminology
+
+- Liquidity Provider : Any external user/company that provides liquidity to VLN by depositing up permitted collateral to mint assets on VLN.
```
Comment (stanly-johnson) [inline pallets/backed_asset/README.md:7]: I just liquidity provider here instead of attestor in the other pallet. Is there a better name?

Comment (olanod) [reply pallets/backed_asset/README.md:7]: Sounds good 👍🏽

File: primitives/README.md (hunk 3 above)
Comment (olanod) [reply primitives/README.md:40]:
```suggestion
This is the default asset of the VLN. USDv is backed by a combination of on-chain and foreign assets pegged to the US Dollar provided by liquidity providers of the VLN.
```

File: primitives/README.md (hunk 2 above)
Comment (olanod) [reply primitives/README.md:23]: For now yes, it was there for demo purposes but is currently not needed. The USDv in this moment exists as some USDC in an external account that Valiu controls, Valiu as a trusted liquidity provider will "attest" it has USDC as a foreign asset and use that to mint USDv. Is up to Valiu to do this manually or automate it based on how its USDC stock changes.

**Review by olanod — APPROVED** [review]

**Pull request merged**